### ISSUE TITLE
Parton shower weights in P8.306 different with respect to P8.240 

### ISSUE THREAD

**Issue by agrohsje** [cms-sw/cmssw#39692]
It seems we are still having a bug in the parton shower weights of P8.3. 
You can see the comparison for isr down between P8.240 and the P8.306 here: 
![image](https://user-images.githubusercontent.com/9880717/194947268-4bebd04a-95e5-45b1-b434-b7cfc6d37cb7.png)
I checked standalone with a few events, and already there the trend seems to be confirmed. 
For FSR however, I couldn't spot at difference. FSR seems to agree reasonably well. 
@smrenna @SanghyunKo @SiewYan @Saptaparna @menglu21

**Comment by makortel** [comment]
assign generators

**Comment by Saptaparna** [comment]
Also opened a pythia issue: https://gitlab.com/Pythia8/releases/-/issues/214

**Comment by agrohsje** [comment]
The last one I can find is 211 : 
https://gitlab.com/Pythia8/releases/-/issues/?sort=created_date&state=all&first_page_size=20
Is it hidden ?

**Comment by agrohsje** [comment]
Hi @makortel. I played around with the "assign generators" command (last version above has no s, but the first one had) but the bot didn't react. As a regular user, one cannot give commands to the bot at all?

**Comment by makortel** [comment]
> As a regular user, one cannot give commands to the bot at all?

Correct, the set of users whose commands the bot accepts is restricted to O&C L1s, release managers, L2s, and specific users listed in
https://github.com/cms-sw/cms-bot/blob/master/categories.py

**Comment by Saptaparna** [comment]
assign generators

**Comment by Saptaparna** [comment]
I can't self assign either.

**Comment by makortel** [comment]
The issue was already assigned to generators.
